spec/controllers/ems_common_controller_spec.rb

```diff
@@ -328,15 +328,6 @@ def test_setting_few_fields
         EvmSpecHelper.create_guid_miq_server_zone
       end
 
-      it "when edit is pressed for unsupported network manager type" do
```

Comment:
Not sure about this one, do we have any other cloud provider with network management still unsupported? ping @Ladas

Reply:
I believe it's just google that doesn't have a network manager